tcconfig/shaper/htb.py

```diff
@@ -112,10 +132,25 @@ def _add_rate(self):
         ]
 
         if bandwidth != upper_limit_rate:
+            # This sets the burst/cburst values to the default tc tries to set,
+            # but works around a bug in tc where values are rounded incorrectly.
+            
+            # Note that "tc class show" will still show burst that are too small,
+            # due to the inverse bug, but if you use "tc -r class show" you will see
+            # the actual size of the cbuffer in bytes.
+            mtu = 1600
```

Comment:
MTU may vary depending on the environment.
Would a fixed value of 1600 be appropriate?

Reply:
This is the value that tc sets as its default when it does this calculation.  It's overridable from the command line, but tcconfig doesn't override it.  Maybe I should also add it as a parameter for tcconfig?

Reply:
It looks like it's necessary to add MTU as a `tcconfig` parameter.
The fix in this PR basically reduces the burst, so it seems like it will have a particularly large impact in environments that use jumbo frames.
I can't accurately predict how much of an impact it will actually have, but adding MTU as an argument seems like it will help reduce the impact in such environments.